examples/simple_import/Makefile

```
@@ -0,0 +1,10 @@
+a.out: main.cpp generated.h src/generated.rs src/lib.rs ../../target/release/libexample_simple_import.a
+	${CXX} -Werror main.cpp generated.cpp -g -L ../../target/release/ -l example_simple_import
+
+../../target/release/libexample_simple_import.a: FORCE
+	cargo build --release
+
+generated.h generated.cpp ./src/generated.rs: main.zng
+	cd ../../zngur-cli && cargo run g ../examples/simple_import/main.zng
+
+FORCE: ;
```

The `.PHONY` target is the proper way to force rebuilds in makefiles

```suggestion
.PHONY: ../../target/release/libexample_simple_import.a

a.out: main.cpp generated.h src/generated.rs src/lib.rs ../../target/release/libexample_simple_import.a
	${CXX} -Werror main.cpp generated.cpp -g -L ../../target/release/ -l example_simple_import

../../target/release/libexample_simple_import.a:
	cargo build --release

generated.h generated.cpp ./src/generated.rs: main.zng
	cd ../../zngur-cli && cargo run g ../examples/simple_import/main.zng
```